Assets/GameAssets/Boss/BossScripts/Attacks/Vine/VineAttack.cs

```diff
@@ -0,0 +1,132 @@
+/*****************************************************************************
+// File Name :         VineAttack.cs
+// Author :            Ryan Swanson
+// Creation Date :     11/7/2024
+//
+// Brief Description : Controls the functionality for the bosses vine attack
```

Comment:
Update the description on `VineAttack.cs` and `VineAttackController.cs` so it's clear how they differ